aea/helpers/search/models.py

```diff
@@ -23,10 +23,18 @@
 from abc import ABC, abstractmethod
 from copy import deepcopy
 from enum import Enum
+from math import asin, cos, radians, sin, sqrt
 from typing import Any, Dict, List, Optional, Type, Union
+from oef.Wrappers import Location as OefLocation
 
-ATTRIBUTE_TYPES = Union[float, str, bool, int]
-SUPPORTED_TYPES = {"str": str, "int": int, "float": float, "bool": bool}
+ATTRIBUTE_TYPES = Union[float, str, bool, int, OefLocation]
+SUPPORTED_TYPES = {
+    "str": str,
+    "int": int,
+    "float": float,
+    "bool": bool,
+    "location": OefLocation,
```

Comment:
needs to be called `Location` not `OefLocation`!

Reply:
If I change it to `Location` it throws an unresolved error.

Reply:
Then you are doing something wrong. You need to use the internal AEA model here and not the OEF model, that's the whole point of introducing the two.